<launch>

    <node type="one_ctrl_node.py" pkg="one_easy_protocol_pkg" name="one_ctrl_node" output="screen" />
    <node type="pick_and_place_node.py" pkg="visy_sorting_app_pkg" name="pick_and_place_node" output="screen" />
    <node type="sorting_app_node.py" pkg="visy_sorting_app_pkg" name="sorting_app_node" output="screen" />

    <node type="grasp_planner_node.py" pkg="visy_sorting_app_pkg" name="grasp_planner_node" output="screen">
      <param name="robot_distance" value="580"/>
      <param name="roi_min" value="50"/>
      <param name="roi_max" value="300"/>
    </node>

</launch>
